ledger/src/blockstore_processor.rs

```diff
@@ -327,42 +333,64 @@ fn execute_batches(
         .flat_map(|batch| batch.sanitized_transactions().to_vec())
         .collect::<Vec<_>>();
 
-    let cost_model = CostModel::new();
     let mut minimal_tx_cost = u64::MAX;
     let mut total_cost: u64 = 0;
+    let mut total_cost_without_bpf: u64 = 0;
     // Allowing collect here, since it also computes the minimal tx cost, and aggregate cost.
     // These two values are later used for checking if the tx_costs vector needs to be iterated over.
+    // The collection is a pair of (full cost, cost without estimated-bpf-code-costs).
     #[allow(clippy::needless_collect)]
     let tx_costs = sanitized_txs
         .iter()
         .map(|tx| {
-            let cost = cost_model.calculate_cost(tx).sum();
+            let tx_cost = cost_model.calculate_cost(tx);
+            let cost = tx_cost.sum();
+            let cost_without_bpf = cost.saturating_sub(tx_cost.bpf_execution_cost);
```

Comment:
May be worth to add a new function to cost_model to calculate cost_without_bpf

Reply:
sum_without_bpf() ?

Reply:
sounds good